docs/guides/agents.md

```diff
@@ -0,0 +1,50 @@
+---
+tags:
+  - LLM
+  - Agents
+---
+# Agents
+
+## Overview
+
+This page provides an overview of different agent frameworks and types for Large Language Models (LLMs), as well as common agent design patterns.
+
+## Agent Frameworks
+
+- LangChain (Python/JavaScript)
```

Comment:
I would go with LangGraph here. Langchain is more of a general framework rather than an agent focused one.